apps/cowswap-frontend/src/modules/swap/containers/SwapWidget/addIntermediateToken.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode, useCallback } from 'react'
+
+import { TokenWithLogo } from '@cowprotocol/common-const'
+
+import { useAddTokenImportCallback } from 'modules/tokensList/hooks/useAddTokenImportCallback'
+import { ImportTokenItem } from 'modules/tokensList/pure/ImportTokenItem'
```

Comment:
We don't usually import out of module index.
Anyway, maybe it's better to make this component part of `modules/tokensList`? Since it mostly uses hooks/components from the module